src/audio/google/google_rtc_audio_processing.c

```diff
@@ -136,31 +136,6 @@ void GoogleRtcFree(void *ptr)
 	return rfree(ptr);
 }
 
-#if CONFIG_IPC_MAJOR_4
-static void google_rtc_audio_processing_params(struct processing_module *mod)
-{
```

Comment:
@singalsu thoughts here?

Reply:
I've added similar function to this all processing functions to have buffer format info set before first copy starts. Depending on component location in pipelines I've seen the copy to start with invalid format. This has been an IPC4 specific issue, could well be that this is no more needed.

Reply:
> Depending on component location in pipelines I've seen the copy to start with invalid format

Yeah, here too.  See the workaround patch at the end of the current series.  We need to get this sorted out; it's not the job of a component to fix its connected buffers.  I'll file bugs.